MEME/MEME/Model/ModelHomeView/View/ModelHomeViewController.swift

```diff
@@ -0,0 +1,403 @@
+    
+    // MARK: - makeConstraints
+    func makeConstraints() {
+        scrollView.snp.makeConstraints { (make) in
+            make.edges.equalTo(view.safeAreaLayoutGuide)
+        }
+        contentsView.snp.makeConstraints { (make) in
+            make.edges.equalTo(scrollView)
+            make.width.equalTo(scrollView)
+        }
+        memeLogoImageView.snp.makeConstraints {make in
+            make.top.equalTo(contentsView.snp.top).offset(10)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+            make.height.equalTo(42)
+            make.width.equalTo(memeLogoImageView.snp.height).multipliedBy(67.0/42.0)
+        }
+        alarmImageView.snp.makeConstraints {make in
+            make.centerY.equalTo(memeLogoImageView.snp.centerY)
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+            make.height.equalTo(22)
+            make.width.equalTo(memeLogoImageView.snp.height).multipliedBy(6.0/7.0)
+        }
+        searchMakeup.snp.makeConstraints {make in
+            make.top.equalTo(memeLogoImageView.snp.bottom).offset(27)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+            make.height.equalTo(41)
+        }
+        modelWelcomeLabel.snp.makeConstraints {make in
+            make.top.equalTo(searchMakeup.snp.bottom).offset(33)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+        }
+        modelWelcomeGuideLabel.snp.makeConstraints {make in
+            make.top.equalTo(modelWelcomeLabel.snp.bottom)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+        }
+        viewAllReservationsButton.snp.makeConstraints {make in
+            make.top.equalTo(modelWelcomeGuideLabel.snp.bottom).offset(17)
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+        }
+        collectionView.snp.makeConstraints {make in
+            make.top.equalTo(viewAllReservationsButton.snp.bottom).offset(13)
+            make.height.equalTo(800)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+            make.bottom.equalTo(contentsView.snp.bottom)
+    
+        }
+    }
+    
+    // MARK: - BindViewModel
+    private func bindViewModel() {
+        let memberId = KeyChainManager.loadMemberID()
+        
+        let input = ModelHomeViewModel.Input(
+            modelReservationsTrigger: Observable.just(memberId),
+            getRecommendArtistByReviewTrigger: .just(()),
+            getRecommendArtistByRecentTrigger: .just(())
+        )
+        
+        let output = viewModel.transform(input: input)
+        
+        Observable.combineLatest(
+            output.modelReservations,
+            output.recommendArtistByReview,
+            output.recommendArtistByRecent
+        ).observe(on: MainScheduler.instance)
```

Comment:
```suggestion
            output.recommendArtistByRecent)
           .observe(on: MainScheduler.instance)
```
마찬가지로 위와 같이 사용하는게 바람직하다고 합니다!